scss/components/_soe_dm_collection.scss

```diff
@@ -133,6 +133,58 @@
   }
 }
 
+//
+// A DIGITAL MAGAZINE ARTICLE COLLECTION BLOCKS
+//
+
+// A ARTICLE COLLECTION BLOCK
+
+.node-type-stanford-magazine-article {
+  .view-stanford-magazine-article-collection {
+    padding-bottom: 40px;
+    padding-top: 40px;
+    background-color: #e4e4e4;
```

Comment:
Please check with @kerri-augenstein , we should use the [pre-defined variables](https://github.com/SU-SOE/stanford_soe_helper/blob/7.x-2.x/scss/variables/_soe_colors.scss). I'm thinking this should be $light-gray